src/components/ButtonPanel.js

```diff
@@ -1,63 +1,50 @@
-            {this.renderButtons('+/-', `${reColor}`)}
-            {this.renderButtons('%', `${reColor}`)}
-            {this.renderButtons('÷')}
-          </div>
-          <div className="number-row">
-            {this.renderButtons('7', `${reColor}`)}
-            {this.renderButtons('8', `${reColor}`)}
-            {this.renderButtons('9', `${reColor}`)}
-            {this.renderButtons('x')}
-          </div>
-          <div className="number-row">
-            {this.renderButtons('4', `${reColor}`)}
-            {this.renderButtons('5', `${reColor}`)}
-            {this.renderButtons('6', `${reColor}`)}
-            {this.renderButtons('-')}
-          </div>
-          <div className="number-row">
-            {this.renderButtons('1', `${reColor}`)}
-            {this.renderButtons('2', `${reColor}`)}
-            {this.renderButtons('3', `${reColor}`)}
-            {this.renderButtons('+')}
-          </div>
-          <div className="number-row">
-            {this.renderButtons('0', `${reColor}`, true)}
-            {this.renderButtons('.', `${reColor}`)}
-            {this.renderButtons('=')}
-          </div>
+const ButtonPanel = ({ clickHandler }) => {
+  const reColor = 'transparent';
+  const handleClick = clickedBtn => clickHandler(clickedBtn);
+  return (
+    <div className="button-panel-container">
+      <div className="number-panel">
+        <div className="number-row">
+          <Button clickHandler={handleClick} value="AC" color={reColor} />
+          <Button clickHandler={handleClick} value="+/-" color={reColor} />
+          <Button clickHandler={handleClick} value="%" color={reColor} />
+          <Button clickHandler={handleClick} value="÷" />
+        </div>
+        <div className="number-row">
+          <Button clickHandler={handleClick} value="7" color={reColor} />
+          <Button clickHandler={handleClick} value="8" color={reColor} />
+          <Button clickHandler={handleClick} value="9" color={reColor} />
+          <Button clickHandler={handleClick} value="x" />
+        </div>
+        <div className="number-row">
+          <Button clickHandler={handleClick} value="4" color={reColor} />
+          <Button clickHandler={handleClick} value="5" color={reColor} />
+          <Button clickHandler={handleClick} value="6" color={reColor} />
+          <Button clickHandler={handleClick} value="-" />
+        </div>
+        <div className="number-row">
+          <Button clickHandler={handleClick} value="1" color={reColor} />
+          <Button clickHandler={handleClick} value="2" color={reColor} />
+          <Button clickHandler={handleClick} value="3" color={reColor} />
+          <Button clickHandler={handleClick} value="+" />
+        </div>
+        <div className="number-row">
+          <Button clickHandler={handleClick} value="0" color={reColor} wide />
+          <Button clickHandler={handleClick} value="." color={reColor} />
+          <Button clickHandler={handleClick} value="=" />
```

Comment:
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.